Libraries/Components/TextInput/TextInput.js

```diff
@@ -130,6 +130,14 @@ export type EditingEvent = SyntheticEvent<
   |}>,
 >;
 
+// macOS-only // [TODO(macOS GH#774)
+export type StateChangeEvent = SyntheticEvent<
```

Comment:
SettingChangeEvent maybe? StateChange feels very generic

Reply:
That sounds much better!